tests/test_umi.yml

```diff
@@ -4,35 +4,61 @@
     - umi
   files:
     - path: results/bam2fastq/test-test_L1_interleaved.fq.gz
+      md5sum: 555d05c18bb10be7c8f51bca559cd9ce
     - path: results/bwamem1/test-test_L1.umi_unsorted.bam
+      md5sum: f037bdc824af9235d9cdbcddfd5ea587
     - path: results/callumiconsensus/test-test_L1_umi-consensus.bam
+      md5sum: c7de733c479032ba670b09db5e5a91fc
     - path: results/cat/test-test_L1_1.merged.fastq.gz
+      md5sum: 5ef818888a8434a0177c306551dc7f0a
     - path: results/cat/test-test_L1_2.merged.fastq.gz
+      md5sum: 6f6c44743064b4d0f8df4764441ad5e6
     - path: results/collate/test-test_L1.mapped_1.fq.gz
+      md5sum: 709872fc2910431b1e8b7074bfe38c67
     - path: results/collate/test-test_L1.mapped_2.fq.gz
+      md5sum: 709872fc2910431b1e8b7074bfe38c67
     - path: results/collate/test-test_L1.mapped_other.fq.gz
+      md5sum: 709872fc2910431b1e8b7074bfe38c67
     - path: results/collate/test-test_L1.mapped_singleton.fq.gz
+      md5sum: 709872fc2910431b1e8b7074bfe38c67
     - path: results/collate/test-test_L1.unmapped_1.fq.gz
+      md5sum: 0ae1b8aa42de8fbb932605a54051941f
     - path: results/collate/test-test_L1.unmapped_2.fq.gz
+      md5sum: 57648df503a9f7cf4da73050c81b9fbc
     - path: results/collate/test-test_L1.unmapped_other.fq.gz
+      md5sum: 709872fc2910431b1e8b7074bfe38c67
     - path: results/collate/test-test_L1.unmapped_singleton.fq.gz
+      md5sum: 709872fc2910431b1e8b7074bfe38c67
```

Comment:
Maybe that's just the way it is, but I was wondering why the checksums are the same for `unmapped` and `mapped`. Can you double check please that it's not a copy & paste error.

Reply:
The file `test-test_L1.unmapped_singleton.fq.gz` consists of just 28 bytes, and zero bytes when decompressed.

There can be no copy-paste error of the md5-sums as all the CI-tests very passing at the time of the review.

Reply:
Okay, so do I understand this correctly that the compressed `fq.gz` files contain more bytes their uncompressed versions? And are all the files with a checksum of `709872fc2910431b1e8b7074bfe38c67` empty then?

Reply:
Yeah, compressing an empty file results in a non-empty file. I guess the (de)compression algorithm requires some "metadata" to be stored in the compressed file.

All the compressed files with md5sum `709872fc2910431b1e8b7074bfe38c67` are empty when decompressed.

Reply:
to be expected when all reads are either R1/R2 and partnered up